README.md

```diff
@@ -20,6 +20,8 @@ The default helmfile is `helmfile.yaml`:
 repositories:
   - name: roboll
     url: http://roboll.io/charts
+    cert-file: optional_client_cert
```

Comment:
Super nit but I slightly prefer `certFile` and `keyFile` as idiomatic yaml object keys seen around the K8S world!
Would you mind if I changed these keys after merging?